test/colony-network-auction.js

```diff
@@ -205,7 +226,8 @@ contract("ColonyNetworkAuction", accounts => {
 
   describe("when bidding", async () => {
     it("can bid", async () => {
-      await giveUserCLNYTokens(colonyNetwork, BIDDER_1, "1000000000000000000");
```

Comment:
I don't get why is this replaced by recreating the colony and then using `bootstrap` to give user tokens.

Reply:
Ok, recreating the colony is definitely not needed, but as we talked, we are minting a very large amount somewhere in the test (~`10**54`). Since maximum rate is ~`3.4e38` we would need to forward time 1 billion years to be able to mint `10**54`, and ganache just breaks with that amount for some reason.

Reply:
How did you calculate the ~3.4e38 maximum rate? I am getting different values? Is that per second, per month?

Reply:
Its per month, the maximum value we can pass to `setTokenIssuanceRate`

Reply:
I am getting 244894656421309 years which is still a very large number to work with considering 10^54 is probably a reasonable amount of tokens people might want minted. After all `TokenIssuanceRate.amount` might need to be a `uint256`?

Reply:
The max `uint128` is `131281777361473172632474771385713` which should be the max issuance rate we pass right?

Reply:
For `uint128` i'm getting => https://www.google.rs/search?q=2+**+128&oq=2+**+12&aqs=chrome.0.69i59j69i57j6j69i59.2522j0j7&sourceid=chrome&ie=UTF-8

If we change `TokenIssuanceRate.amount` to `uint256` there is possible overflow [here](https://github.com/JoinColony/colonyNetwork/pull/289/files#diff-669f459dea66cccc363b550a82126c97R167) and [here](https://github.com/JoinColony/colonyNetwork/pull/289/files#diff-669f459dea66cccc363b550a82126c97R179)

Reply:
Yes you're right, that's per month and mine is per second.

Reply:
So is `3.4e38` enough as upper limit?

Reply:
Let's leave it like that for now